### PR TITLE
taskForms

### PR DESCRIPTION
**Pull request by vladshavel**
Can you check please
commit Forms1
commit" NewForm" its your demo(

### CODE REVIEW

File: vshavel/forms.html [hunk 1]
```diff
@@ -0,0 +1,209 @@
...
+				</fieldset>
+			</td>
+			<td>
+				<h3>Валидация данных HTML5:</h3>
+				<form method="get" action="http://sc.it-academy.by/fe/form.php">
+					<fieldset>
+						<legend>Строка поиска</legend>
+						<input type="text" name="tex2" placeholder="Только русские буквы" pattern="[А-я]+"><br>
+						<span>Телефонный номер формата:</span><br>
+						<input type="text" name="phone" placeholder="+XXX-XX-XXXXXXX" pattern="\+375-(29|44|25)-[0-9]{7}"><br>
```
Comment (pluhin) [inline vshavel/forms.html]: Wrong pattern

File: vshavel/site.by/index.html [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+         <ul>
+                 <li><a href="./index.html">Главная</a></li>
+                 <li><a href="./catalog.html">Каталог</a></li>
+                 <li><a href="./promo.html">Акции</a></li>
+                 <li><a href="./about.html">О нас</a></li>
+         </ul>
+		 
+        <h2>Меню с вложенным спском</h2>
+		
+          <ul>
```
Comment (pluhin) [inline vshavel/site.by/index.html:25]: Wrong hierarchy for nested lists

**Review by pluhin — CHANGES_REQUESTED** [review]

File: vshavel/site.by/index.html (hunk 2 above)
Comment (vladshavel) [reply vshavel/site.by/index.html:25]: What’s wrong here?

File: vshavel/forms.html [hunk 3]
```diff
@@ -0,0 +1,209 @@
...
+				<input type="checkbox" name "check1" id="check1">Оставаться в системе
+			</label><br>
+			<input type="button" name="enter" value="Войти">
+				</fieldset>
+			</form>
+		</td>
+		<td>
+			<h3>Опрос(голосование)</h3>
+			<form method="post" action="http://sc.it-academy.by/fe/form.php">
+			<label>
```
Comment (pluhin) [inline vshavel/forms.html]: Why did you open label here?

File: vshavel/forms.html [hunk 4]
```diff
@@ -0,0 +1,209 @@
...
+				<fieldset>
+					<legend>Как вам?</legend>
+					
+					<input type="radio" name="radio_group"
+					value="radio1" id="radgr1" checked> Плохо
+			</label><br>
+			<label>
+					<input type="radio" name="radio_group
```
Comment (pluhin) [inline vshavel/forms.html]: value of attribute wasn't closed (the quote is missing)

File: vshavel/forms.html [hunk 5]
```diff
@@ -0,0 +1,209 @@
...
+					value="radio2" id"=radgr2"> Неплохо
+			</label><br>
+			<label>
+					<input type="radio" name="radio_group
```
Comment (pluhin) [inline vshavel/forms.html]: value of attribute wasn't closed (the quote is missing)

File: vshavel/forms.html [hunk 6]
```diff
@@ -0,0 +1,209 @@
...
+					value="radio3 id"=radgr3"> Хорошо
+			</label><br>	
+			<label>
+					<input type="radio" name="radio_group
```
Comment (pluhin) [inline vshavel/forms.html]: value of attribute wasn't closed (the quote is missing)

File: vshavel/forms.html [hunk 7]
```diff
@@ -0,0 +1,209 @@
...
+
+			</fieldset>
+		</form>
+		</td>
+		<td>
+			<h3>Выпадающий и мульти списки:</h3>
+			
+				<fieldset>
+					<legend>Города РБ</legend>
+					<form method="post" action="http://sc.it-academy.by/fe/form.php">
```
Comment (pluhin) [inline vshavel/forms.html]: form should be open in line 141

File: vshavel/forms.html [hunk 8]
```diff
@@ -0,0 +1,209 @@
...
+							</select>
+						</fieldset>
+					</form>
+				</td>
+			</tr>
+			<tr>
+				<td>
+					<h3>Загрузка файлф на сервер</h3>
+			
+					<fieldset>
```
Comment (pluhin) [inline vshavel/forms.html:181]: filed set should be inside form tag,  usually

File: vshavel/site.by/index.html (hunk 2 above)
Comment (pluhin) [reply vshavel/site.by/index.html:25]: Example:
Wrong section:
```
 <li><a href="./catalog.html">Каталог</a></li>
              <ul>
                        <li><a href="./catalog/a_w.html">Осень-Зима</a></li>
                        <li><a href="./catalog/s_s.html">Весна-Лето</a></li>
              </ul>

Correct section:
 <li><a href="./catalog.html">Каталог</a>
              <ul>
                        <li><a href="./catalog/a_w.html">Осень-Зима</a></li>
                        <li><a href="./catalog/s_s.html">Весна-Лето</a></li>
              </ul>
</li>
```

**Review by pluhin — CHANGES_REQUESTED** [review]

File: vshavel/site.by/index.html (hunk 2 above)
Comment (vladshavel) [reply vshavel/site.by/index.html]: there are list menu
and another list menu with another lists
i checked it and everything ok

File: vshavel/forms.html (hunk 4 above)
Comment (vladshavel) [reply vshavel/forms.html]: can you tell me what do you mean?

**Review by pluhin — CHANGES_REQUESTED** [review]
Not all remarks have been changed, see comments in forms file

**Comment by vladshavel** [comment]
not everything outdated but i think that i did all remarks

File: vshavel/forms.html (hunk 5 above)
Comment (vladshavel) [reply vshavel/forms.html]: i closed all atributes

Comment (pluhin) [reply vshavel/forms.html]: Not all 
name="radio_group   <-  here missing "

**Review by pluhin — APPROVED** [review]
Good, accepted

**Pull request merged**